README.md

```diff
@@ -1,5 +1,10 @@
 ## HIDAPI library for Windows, Linux, FreeBSD and macOS
 
+| Build type           | Status |
+|----------------------|--------|
+| `macOS master`       | [![Build Status](https://travis-ci.org/libusb/hidapi.svg?branch=master)](https://travis-ci.org/libusb/hidapi) |
+| `Linux/BSD, last PR` | [![builds.sr.ht status](https://builds.sr.ht/~qbicz/hidapi.svg)](https://builds.sr.ht/~qbicz/hidapi?) |
```

Comment:
> last PR

this is not very accurate

"last build (branch/PR)" - accurate, but too verbose (and long)

maybe some other suggestion?

Reply:
The accurate description looks best here, thanks!